subset/security/Dockerfile.test_tls

```diff
@@ -1,9 +1,9 @@
 FROM daq/aardvark:latest
 
 # Do this alone first so it can be re-used by other build files.
-RUN $AG update && $AG install openjdk-8-jre
+RUN $AG update && $AG install openjdk-11-jre
```

Comment:
Is jdk-11 actually required?  Because of the way Docker works, this step then will no longer be optimized with other system.  Not fatal, but would unnecessarily increase build time.

Reply:
Yes, JDK 8 does not support TLS 1.3.

Reverted to jdk 8 to confirm this and reran tests so that I could recreate the error 

> Exception main:TLSv1.3 SSLContext not available